src/shared/port-rpc.js

```diff
@@ -51,39 +51,63 @@ const PROTOCOL = 'frame-rpc';
  */
 
 /**
- * RPC provides remote procedure calls between frames.
+ * PortRPC provides remote procedure calls between frames or workers. It uses
+ * the Channel Messaging API [1] as a transport.
  *
- * It uses the Channel Messaging API [1] for inter-frame communication.
+ * To communicate between two frames with this class, construct a PortRPC
+ * instance in each and register method handlers with {@link on}. Create a
+ * {@link MessageChannel} and send one of its two ports to each frame. Then call
+ * {@link connect} to make the PortRPC instance in each frame use the corresponding
+ * port.
  *
  * [1] https://developer.mozilla.org/en-US/docs/Web/API/Channel_Messaging_API
  *
- * Code adapted from https://github.com/substack/frame-rpc.
```

Comment:
The copyright note for frame-rpc is still at the top of this file, although the code has evolved so much that it has only small connections to this package.